.env

```diff
@@ -1,5 +1,5 @@
-SKIP_PREFLIGHT_CHECK=true
-PROXY_TARGET=http://senderfsp-pm4ml.qa.prod-dev.mbox-dev.io:20000
-PROXY_PORT=10000
-PROXY_API_PATH=/experience-api
-PORT=8081
+SKIP_PREFLIGHT_CHECK=true
+PROXY_TARGET=http://senderfsp-pm4ml.qa.prod-dev.mbox-dev.io:20000
+PROXY_PORT=10000
+PROXY_API_PATH=/experience-api
+PORT=5500

```

Comment:
Please do not change the default values there. It breaks the local development process for somebody else.

Reply:
can I add .env to gitignore so that my local configs doesn't get pushed?

Reply:
Yes I think that's the way to go